src/renderers/shared/ReactDebugTool.js

```diff
@@ -192,6 +192,63 @@ function resumeCurrentLifeCycleTimer() {
   currentTimerType = timerType;
 }
 
+var lastMarkTimeStamp = null;
+var canUsePerformanceMeasure =
+  typeof performance !== 'undefined' &&
+  typeof performance.mark === 'function' &&
+  typeof performance.clearMarks === 'function' &&
+  typeof performance.measure === 'function' &&
+  typeof performance.clearMeasures === 'function';
+
+function shouldMark(debugID) {
+  if (!isProfiling || !canUsePerformanceMeasure) {
```

Comment:
Is there any problem with doing them? In my experience of using standards, unless you check for everything explicitly, you risk running into bad/incomplete polyfills.